app/uk/gov/hmrc/minorentityidentificationfrontend/featureswitch/core/config/FeatureSwitchingModule.scala

```diff
@@ -34,7 +34,7 @@ class FeatureSwitchingModule extends Module with FeatureSwitchRegistry {
   }
 }
 
-case object OverseasVerificationStub extends FeatureSwitch {
-  override val configName: String = "feature-switch.overseas-further-verification-stub"
-  override val displayName: String = "Use stub for Overseas Company further verification flow"
-}
\ No newline at end of file
+case object TrustVerificationStub extends FeatureSwitch {
+  override val configName: String = "feature-switch.trust-verification-stub"
+  override val displayName: String = "Use stub for Trust verification flow"
```

Comment:
Maybe rename to something like 'Enable full trust journey' to better describe what it will do